packages/core/core/src/config.ts

```diff
@@ -198,7 +201,8 @@ export class Config {
       context: this.context,
       env: this.env,
       version: this.version,
-      client: this.client
+      client: this.client,
+      webpackConfigName: this.webpackConfigName
```

Comment:
No need to pass the `webpackConfigName` to the plugins, then to the plugn. As soon as it is part of `config`, it is accessible as `embark.config.webpackConfigName`. It also makes more sense to check it in configs than inside the plugin itself IMO.